_data/projects/hyperswitch.yml

```diff
@@ -15,6 +15,9 @@ upforgrabs:
   name: hacktoberfest
   link: >-
     https://github.com/juspay/hyperswitch/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest+no%3Aassignee
+  name: "good first issue"
```

Comment:
Sadly this syntax isn't currently supported for multiple labels - this will likely just overwrite the previous value, if I understand YAML correctly...